examples/JWT.py

```diff
@@ -0,0 +1,14 @@
+from plivo.utils import jwt
+import time
+
+# using valid_from in epoch and lifetime in seconds
+token = jwt.AccessToken('MADADADADADADADADADA', 'qwerty', 'username', valid_from=time.time(), lifetime=300, uid='username-12345')
```

Comment:
we should keep example same across all SDKs.
here also, we can use:
'{authId}', '{authToken}', 'endpointUsername', 'uid'

or

auth_id='MADADADADADADADADADA', auth_token='qwerty' .....etc.